arm-relay/2016-07-01/relay.json

```diff
@@ -1991,8 +2109,21 @@
         }
       },
       "description": "Parameters supplied to the Regenerate Auth Rule."
+    },
+    "ErrorResponse": {
+      "description": "Error reponse indicates CDN service is not able to process the incoming request. The reason is provided in the error message.",
```

Comment:
Completely sloppy job at doing things. Blind copy paste from the CDN service. 
@sazeesha - Please take a look at this.